verification/src/http_server/handlers/verification/solidity/types.rs

```diff
@@ -23,6 +28,46 @@ pub struct MultiPartFiles {
     contract_libraries: Option<BTreeMap<String, String>>,
 }
 
+impl Apiv2Schema for MultiPartFiles {
```

Comment:
`MultiPartFiles` has `sources: BTreeMap<PathBuf, String>`;
`Apiv2Schema` is not implemented for `PathBuf`, so we have to do our own `impl Apiv2Schema for MultiPartFiles`